openhands-cli/openhands_cli/locations.py

```diff
@@ -7,3 +7,5 @@
 WORK_DIR = os.getcwd()
 
 AGENT_SETTINGS_PATH = "agent_settings.json"
+
+MCP_CONFIG_POINTER_FILE = "mcp_config_pointer.txt"
```

Comment:
I'm thinking.. we should just ask user to put mcp.json under `~/.openhands` instead maintaining a pointer here, wdyt?